ifrs17-template/InputFormatDescription.ipynb

```diff
@@ -74,13 +74,13 @@
             "source": [
                 "An Actuals file is named **Actuals** *\\_<reportingNode\\>* *\\_<year\\>* *\\_<month\\>* and is composed of two sections: **Main** and **Actual**.",
                 "\n",
-                "\nThe **Main** section contains information such as: Reporting Node, Year, Month for which the data is being imported. This information is used to define the [partition](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/DataModel/DataStructure#partition) to which the data belong.",
+                "\nThe **Main** section contains information such as: Reporting Node, Year, Month for which the data is being imported. This information is used to define the [partition](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/DataModel/DataStructure#partition) to which the data belong.",
```

Comment:
Typo? data is being imported --> data are being imported

Reply:
Thanks, both constructions are standard: https://www.merriam-webster.com/dictionary/data. Given we mostly use it as singular I also leave this as is.